CMakeLists.txt

```diff
@@ -1,46 +1,22 @@
-# Catkin User Guide: http://www.ros.org/doc/groovy/api/catkin/html/user_guide/user_guide.html
-# Catkin CMake Standard: http://www.ros.org/doc/groovy/api/catkin/html/user_guide/standards.html
 cmake_minimum_required(VERSION 2.8.3)
 project(dsp3000)
-# Load catkin and all dependencies required for this package
-# TODO: remove all from COMPONENTS that are not catkin packages.
-find_package(catkin REQUIRED COMPONENTS roscpp message_generation std_msgs)
-
-# include_directories(include ${Boost_INCLUDE_DIR} ${catkin_INCLUDE_DIRS})
-# CATKIN_MIGRATION: removed during catkin migration
-# cmake_minimum_required(VERSION 2.4.6)
 
-# CATKIN_MIGRATION: removed during catkin migration
-# include($ENV{ROS_ROOT}/core/rosbuild/rosbuild.cmake)
+find_package(catkin REQUIRED COMPONENTS roscpp message_generation std_msgs)
 
+catkin_package(
+    CATKIN_DEPENDS roscpp std_msgs
+)
 
-# CATKIN_MIGRATION: removed during catkin migration
-# rosbuild_init()
+include_directories(cereal_port/include)
+include_directories(${catkin_INCLUDE_DIRS})
+add_definitions("-std=c++11 -Wall -Wextra -Werror")
```

Comment:
Please use `set_directory_properties` for this, eg: https://github.com/ros/ros_comm/blob/kinetic-devel/clients/roscpp/CMakeLists.txt#L5